src/components/ShareSocialCta/index.js

```diff
@@ -68,7 +68,7 @@ function ShareSocialCta({
                   currentUrl={currentUrl}
                   facebookQuote={facebookQuote}
                   facebookHashtag={facebookHashtag}
-                />
+                >{null}</FacebookButton>
```

Comment:
just out of curiosity, why is it better to wrap the button around `null`?

Reply:
Because just leaving it empty was making the linter complain, and it's better to actually put `null ` for screen readers than have it read an empty string. We had it in a GDS audit

Reply:
Because just leaving it empty was making the linter complain, and it's better to actually put `null ` for screen readers than have it rea

Reply:
thanks for explaining - I'm learning so much! I even used aria-labels today in work after reviewing this PR :D